### PR TITLE
libslirp 4.3.1 (new formula)

### PR DESCRIPTION
**Pull request by RayyanAnsari**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/libslirp.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  sha256 "388b4b08a8cc0996cc5155cb027a097dc1a7f2cfe84b1121496608ab5366cc48"
+  license "BSD-3-Clause"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "sed", "-i", "-e", "s/,--version-script//g", "meson.build"
```
Comment (SMillerDev) [inline Formula/libslirp.rb:14]: Please use `inreplace` and try and patch this upstream as well.

Comment (WaluigiWare64) [reply Formula/libslirp.rb:14]: It was recently fixed upstream. I'll use `inreplace`.

Comment (WaluigiWare64) [reply Formula/libslirp.rb]: resolved.

**Comment by WaluigiWare64** [comment]
Do I need to squash these commits or do  you do it yourselves?

**Review by SMillerDev — APPROVED** [review]
Thanks @WaluigiWare64  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by WaluigiWare64** [comment]
Thanks for all your and other maintainers hard work on making the Homebrew Package Manager   available to all!

**Pull request closed without merging**